app/src/test/java/com/example/a3048_carapp/ExampleUnitTest.kt

```diff
@@ -11,7 +11,7 @@ import org.junit.Assert.*
  */
 class ExampleUnitTest {
     @Test
-    fun addition_isCorrect() {
+    fun `addition is Correct`() {
```

Comment:
I changed the name of a unit test using underscores and double quotes to using backticks and spaces to make the unit test names more readable.
https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods